packages/commerce-sdk-react/src/hooks/useMutation.ts

```diff
@@ -96,61 +85,18 @@ export const useCustomMutation = (
                     }
                 },
                 clientConfig: {
-                    ...currentClientConfig,
+                    parameters: {
+                        clientId: config.clientId,
+                        siteId: config.siteId,
+                        organizationId: config.organizationId,
+                        shortCode: config.organizationId
```

Comment:
shortCode assigned to a org Id value?

Reply:
This is a reverse commit. Seems that was the case prior to this

Reply:
Good catch, @unandyala! Fixed